src/test/java/com/puppycrawl/tools/checkstyle/SuppressionsStringPrinterTest.java

```diff
@@ -0,0 +1,129 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2018 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle;
+
+import static com.puppycrawl.tools.checkstyle.internal.utils.TestUtil.isUtilsClassHasPrivateConstructor;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import antlr.NoViableAltException;
+import com.puppycrawl.tools.checkstyle.api.CheckstyleException;
+
+public class SuppressionsStringPrinterTest extends AbstractTreeTestSupport {
+
+    private static final String LINE_SEPARATOR = System.getProperty("line.separator");
+
+    @Override
+    protected String getPackageLocation() {
+        return "com/puppycrawl/tools/checkstyle/suppressionsstringprinter";
+    }
+
+    @Test
+    public void testIsProperUtilsClass() throws ReflectiveOperationException {
+        assertTrue("Constructor is not private",
+                isUtilsClassHasPrivateConstructor(SuppressionsStringPrinter.class, true));
+    }
+
+    @Test
+    public void testCorrect() throws Exception {
+        final String expected = new String(Files.readAllBytes(Paths.get(
+                getPath("ExpectedSuppressionsStringPrinter.txt"))),
```

Comment:
we put Expected to file only if it is huge to be embedded. I do not see your expected output as huge. If there was some previous discussion .. please give me a link to it to review. If no discussions before, please inline expected to UT method.

Reply:
done